src/pgduckdb_background_worker.cpp

```diff
@@ -86,8 +86,8 @@ static bool reused_bgw_session_hint = false;
  */
 static bool set_up_unclaim_session_hint_hook = false;
 
-static void SyncMotherDuckCatalogsWithPg(bool drop_with_cascade, duckdb::ClientContext &context);
-static void SyncMotherDuckCatalogsWithPg_Cpp(bool drop_with_cascade, duckdb::ClientContext *context);
+void SyncMotherDuckCatalogsWithPg(bool drop_with_cascade, duckdb::ClientContext &context);
+void SyncMotherDuckCatalogsWithPg_Cpp(bool drop_with_cascade, duckdb::ClientContext &context);
```

Comment:
Why this change? It's not used outside of this file right? And it's not in the anonymous namespace now.